netdata.spec.in

```diff
@@ -23,6 +23,11 @@ License:	GPL v3+
 Group:		Applications/System
 Source0:	http://firehol.org/download/netdata/releases/v@PACKAGE_VERSION@/%{name}-@PACKAGE_VERSION@.tar.xz
 URL:		http://my-netdata.io/
+%if 0%{?suse_version}
+BuildRequires:	automake
+BuildRequires:	systemd-rpm-macros
```

Comment:
is this actually required for build? just curios for what.

anyway, we should put some macro in the prologue to remove this conditional.